src/stores.ts

```diff
@@ -1,5 +1,6 @@
-import { getBlocknative } from './services'
+import { Emitter } from 'bnc-sdk/dist/types/src/interfaces'
```

Comment:
@aaronbarnardsound is there a reason why this can't be exported directly from the module?

Reply:
@taylorjdawson We'd need to export them from the sdk module directly which would be a nice improvement for sure